### DISCUSSION TITLE
**Config Validation Error** when load en_core_web_trf

### DISCUSSION THREAD

**Discussion by su-park** [explosion/spaCy#8480] [Help: Other Questions] [answered]
## How to reproduce the behaviour

When I try to load 'en_core_web_trf' it returns below error.

```
import spacy
nlp = spacy.load("en_core_web_trf")
```

```
ConfigValidationError: 

Config validation error

batch_size   field required

{'lang': 'en', 'pipeline': ['transformer', 'tagger', 'parser', 'ner', 'attribute_ruler', 'lemmatizer'], 'disabled': [], 'before_creation': None, 'after_creation': None, 'after_pipeline_creation': None, 'tokenizer': {'@tokenizers': 'spacy.Tokenizer.v1'}}
```

## Your Environment

- **spaCy version:** 3.0.6
- **Platform:** Linux-4.15.0-45-generic-x86_64-with-Ubuntu-18.04-bionic
- **Python version:** 3.7.5
- **Pipelines:** en_core_web_sm (3.0.0), en_core_web_trf (3.0.0a0)

**Comment by adrianeboyd** [comment, marked as answer]
Hi, note that this is an alpha version of the trf model (3.0.0a0 instead of 3.0.0), so it wasn't the final release. Use `spacy download` to download the updated version.

**Reply by su-park** [reply]
Thank you for your swift response.

Since I am working on closed environment there is no way but to download and install the model in [repository(en_core_web_trf-3.0.0.tar.gz
)](https://github.com/explosion/spacy-models/releases/tag/en_core_web_trf-3.0.0) manually.

Is there any way to download updated version?

**Reply by su-park** [reply]
check [3.0.0](https://newreleases.io/project/github/explosion/spacy-models/release/en_core_web_trf-3.0.0).

I guess this is the lastest version.

**Reply by polm** [reply]
Yes, the release you linked to is the latest version. If you download the tar gz you should be able to just install it with `pip install`.
